stacktracer/src/main/scala/zio/internal/stacktracer/TracerUtils.scala

```diff
@@ -9,10 +9,10 @@ private[internal] object TracerUtils {
    * performances reasons. See zio.internal.stacktracer.Tracer.createTrace for
    * the format of the trace
    */
-  def parse(trace: String): Option[(String, String, Int)] = {
+  def parse(trace: String): (String, String, Int) = {
```

Comment:
For the same reasons as above, we should probably keep this method and add another one that is `parseOrNull`